### PR TITLE
Add libpgmath

### PR DESCRIPTION
**Pull request by isuruf**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages
- [ ] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge/staged-recipes , has anyone seen the error below?
```
File "C:/Miniconda/conda-bld/libpgmath_1557464813718/_build_env/Scripts/lit.exe", line 1
SyntaxError: Non-UTF-8 code starting with '\x90' in file C:/Miniconda/conda-bld/libpgmath_1557464813718/_build_env/Scripts/lit.exe on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

`lit.exe` is a console_script from a noarch package.

**Comment by ocefpaf** [comment]
> @conda-forge/staged-recipes , has anyone seen the error below?
> 
> ```
> File "C:/Miniconda/conda-bld/libpgmath_1557464813718/_build_env/Scripts/lit.exe", line 1
> SyntaxError: Non-UTF-8 code starting with '\x90' in file C:/Miniconda/conda-bld/libpgmath_1557464813718/_build_env/Scripts/lit.exe on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
> ```
> 
> `lit.exe` is a console_script from a noarch package.

I've seen similar errors on Windows. Not sure what is the best way to fix it though.

**Comment by isuruf** [comment]
@conda-forge/staged-recipes, ready for review.

This doesn't have a stable release yet, but they use the versioning scheme `yyyy.mm.dd`, which I've used here with the last commit's date.

File: recipes/libpgmath/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "libpgmath" %}
+{% set version = "2019.08.01" %}
```
Comment (scopatz) [inline recipes/libpgmath/meta.yaml]: Is this going to cause problems with future version numbers?

File: recipes/libpgmath/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python
+    - pip
+    - clangxx   # [not osx]
+    # - lit
```
Comment (scopatz) [inline recipes/libpgmath/meta.yaml]: please remove comments

File: recipes/libpgmath/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipes/libpgmath/meta.yaml]: No, they have version numbers like that (last release version number is 2019.03.29), but my PR for windows and osx support landed recently and I want to have this package now to play with it.

File: recipes/libpgmath/meta.yaml (hunk 2 above)
Comment (isuruf) [reply recipes/libpgmath/meta.yaml]: If you don't mind, I'd like to keep this to remind me to fix the `lit` conda package on windows.

Comment (scopatz) [reply recipes/libpgmath/meta.yaml]: Can you add an explanatory comment then, please?

File: recipes/libpgmath/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/libpgmath/meta.yaml]: Ok, sounds good

File: recipes/libpgmath/meta.yaml (hunk 2 above)
Comment (isuruf) [reply recipes/libpgmath/meta.yaml]: Sure. Done.

Comment (scopatz) [reply recipes/libpgmath/meta.yaml]: Thanks!

**Comment by isuruf** [comment]
Thanks

**Pull request merged**